notes-service/src/main/java/org/exoplatform/wiki/model/Page.java

```diff
@@ -98,6 +98,8 @@ public class Page {
   private Page                            parent;
 
   private Map<String, List<MetadataItem>> metadatas;
+  
+  private String                          lastUpdater;
```

Comment:
As agreed in a meeting with @mkrout and @rdenarie we will reuse the `author` instead to have the last modifier, can we use it instead ?